SignalServiceKit/src/Contacts/SignalAccount.m

```diff
@@ -46,6 +47,23 @@ - (nullable SignalRecipient *)signalRecipientWithTransaction:(YapDatabaseReadTra
     return [SignalRecipient recipientWithTextSecureIdentifier:self.recipientId withTransaction:transaction];
 }
 
+- (nullable NSString *)uniqueId
+{
+    return _recipientId;
```

Comment:
I think we might want to check for empty recipientIds wherever we create new instances of `SignalRecipient`.

Reply:
done